File: app/src/main/java/jensklingenberg/de/tuxoid/model/GameImageView.java
package jensklingenberg.de.tuxoid.model;

import android.content.Context;

import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.widget.ImageView;
import jensklingenberg.de.tuxoid.model.element.Element;

/**
 * Created by jens on 22/7/17.
 */

public class GameImageView extends ImageView {

    public GameImageView(Context context, Element bit) {
        super(context);
        setImageBitmap(bit.getImage());

    }

    public GameImageView(Context context) {
        super(context);
    }

    public GameImageView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public GameImageView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public GameImageView(Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }
}

File: app/src/main/java/de/jensklingenberg/tuxoid/model/GameImageView.kt
package de.jensklingenberg.tuxoid.model

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatImageView
import de.jensklingenberg.tuxoid.model.element.Element

/**
 * Created by jens on 22/7/17.
 */
class GameImageView : AppCompatImageView {
    constructor(context: Context?, bit: Element) : super(context) {
        setImageBitmap(bit.image)
    }

    constructor(context: Context?) : super(context) {}
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {}
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {}
}